tasks/support/templates/appliance_installation_script.rb.erb

```diff
@@ -7,22 +7,34 @@ require "bundler/definition"
 
 GEM_INSTALL_DIR          = "/tmp/manageiq-performance-gem-dir"
 
+<% unless @solo_gem -%>
```

Comment:
would it logically make more sense to put this `unless` check around this and the next block?